min_vel_xy: 0.01
min_vel_yaw: 0.05
pid:
  x: {p: 1.0, i: 0.0, d: 0.0}
  y: {p: 1.0, i: 0.0, d: 0.0}
  yaw: {p: 1.0, i: 0.0, d: 0.0}
goal_x: -0.6  # meters ahead of tag
goal_yaw: 0.0  # radians relative to tag frame
error_threshold_xy: 0.05
error_threshold_yaw: 0.05
timeout_xy: 5
timeout_yaw: 5